altapay.php

```diff
@@ -357,193 +355,193 @@ public function getContent()
     public function renderAddForm()
     {
         $currencyOptions = [];
-        $terminalNature  = [];
-        foreach (Currency::getCurrencies((int)Context::getContext()->language->id) as $currency) {
+        $terminalNature = [];
+        foreach (Currency::getCurrencies((int) Context::getContext()->language->id) as $currency) {
             $currencyOptions[] = [
-                'id'   => $currency->iso_code,
-                'name' => $currency->name . ' (' . $currency->iso_code . ')'
+                'id' => $currency->iso_code,
+                'name' => $currency->name . ' (' . $currency->iso_code . ')',
             ];
         }
-        $iconOptions       = [];
-        $fieldsForm        = [];
-        $tokenControl      = [];
-        $directory         = _PS_MODULE_DIR_ . '/' . $this->name . '/' . $this->paymentMethodIconDir;
+        $iconOptions = [];
+        $fieldsForm = [];
+        $tokenControl = [];
+        $directory = _PS_MODULE_DIR_ . '/' . $this->name . '/' . $this->paymentMethodIconDir;
         $scanned_directory = array_diff(scandir($directory), ['..', '.', '.DS_Store']);
         foreach ($scanned_directory as $filename) {
             $iconOptions[] = [
-                'id'   => $filename,
-                'name' => $filename
+                'id' => $filename,
+                'name' => $filename,
             ];
         }
         $ccTokenControlOptions = [
             [
                 'name' => 'Enable',
-                'val'  => 1
-            ]
+                'val' => 1,
+            ],
         ];
-        $terminals             = $this->getAltapayTerminals();
+        $terminals = $this->getAltapayTerminals();
         foreach ($terminals as $terminal) {
             $terminalNature[] = [
-                'id'   => $terminal['nature'],
+                'id' => $terminal['nature'],
                 'name' => $terminal['nature'],
             ];
         }
 
         if (_PS_VERSION_ >= '1.7.0.0') {
```

Comment:
Again this should be using `version_compare`, please search the plugin for more of these incorrect comparisons.